lib/chatbotStateEnum.js

```diff
@@ -4,6 +4,7 @@ const CHATBOT_STATE = {
   RESPONDING: 'RESPONDING',
   WAITING_FOR_CATEGORY: 'WAITING_FOR_CATEGORY',
   COMPLETED: 'COMPLETED',
+  RESET: 'RESET',
```

Comment:
Cool. I agree. Having a separate chatbot state for this is useful in case any clients come back to us questioning why an alert didn't happen.